xtask/src/temp.rs

```diff
@@ -0,0 +1,35 @@
+use std::path::{Path, PathBuf};
```

Comment:
We might not need this. I felt it might be handy to integrate it with our CI scripts, but it's probably unneeded.